net-dns/maradns/maradns-2.0.17.ebuild

```diff
@@ -0,0 +1,111 @@
+# Copyright 2019 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+PYTHON_COMPAT=( python2_7 )
```

Comment:
Is there no chance for python3 support? We are currently looking into dropping python2.